README.md

```diff
@@ -87,6 +87,47 @@ The Ginkgo CUDA module has the following __additional__ requirements:
 The Ginkgo OMP module has the following __additional__ requirements:
 *  _MinGW_ or _Cygwin_
 
+Depending on the configuration settings, some manual work might be required:
+* Build Ginkgo as shared library:
+  Add `PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` into the environment variable `PATH`.
+  `GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH` is `windows_shared_library` by default. More Details are available in the [Installation page](./INSTALL.md).
+  * cmd: `set PATH="<PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH>;%PATH%"`
+  * powershell: `$env:PATH="<PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH>;$env:PATH"`
+
+  CMake will gives the error message including `<PROJECT_BINARY_DIR/GINKGO_WINDOWS_SHARED_LIBRARY_RELPATH>` if the path is not correct.
+  ```Did not find this build in the environment variable PATH. Please add <path> into the environment variable PATH.```
+* Build Ginkgo with Debug mode:
```

Comment:
The following issues can happen but it depends on the machine setup etc? In this case, I would add some context here:

Some `Debug` build specific issues can appear depending on the machine and environment. The known issues are the following: